cohortextractor/query_engines/mssql.py

```diff
@@ -355,10 +364,8 @@ def build_condition_statement(self, comparator, tables):
             connector = getattr(sqlalchemy, comparator.connector)
             condition_statement = connector(left_conditions, right_conditions)
         else:
-            query_node = comparator.lhs
-            table = tables[query_node]
```

Comment:
This `tables` argument is no longer used, I think.

Reply:
:+1: